hathor/manager.py

```diff
@@ -415,7 +415,8 @@ def _initialize_components(self) -> None:
                 else:
                     # TODO: deal with invalid tx
                     if not tx_meta.validation.is_final():
-                        assert tx_meta.validation.is_at_least_basic(), f'invalid: {tx.hash_hex}'
+                        if not tx_meta.validation.is_checkpoint():
```

Comment:
Isn't it better to do this inside `is_at_least_basic()`?

Reply:
I don't think this change adds anything, since a checkpoint-valid tx will always be at least basic-valid, by the definition of the validation states.

Reply:
Fixed in #292 .